dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,129 @@
+# Proposal: Enhance task manageability
+
+- Author(s): [ehco](https://github.com/Ehco1996)
+- Last updated: 2022-01-22
+
+## Background
+
+The main purpose of this change is to address the root cause of problems like [#3771](https://github.com/pingcap/tiflow/issues/3771), which are caused by the fact that DM task itself does not distinguish between dynamic configuration and static resources, making it impossible for users to intuitively manage their tasks. For this reasons, we will attempt to redesign the state machine of DM task and optimize the dmctl interaction interface to provide a better user experience.
+
+### Current State machine of task
+
+![dm old task lifecycle](../media/dm-old-task-lifecycle.png)
+
+### New State machine of task
+
+![dm new task lifecycle](../media/dm-new-task-lifecycle.png)
+
+This is mainly achieved by adding a new command `dmctl task create` , which creates a stopped task instead of creating and starting a new task with single command `dmctl start-task`.
+
+## Goals
+
+- Organize and optimize the state machine of tasks for better management
+- Unified style of commands in dmctl, optimized dmctl interaction experience
+- Adding/modifying OpenAPI, enhance eco-tools
+
+## Design and Examples
+
+New syntax of dmctl is `dmctl [resource type] [command] [flags]`
+
+where `resource type`, `command` and `flags` are:
+
+- `resource type`: Specifies the resource you want to control. resource types are case-insensitive and there are limited resource types.
```

Comment:
```suggestion
- `resource type`: Specifies the resource you want to control. Resource types are case-insensitive and there are limited resource types.
```

> there are limited resource types

I'm expecting to read the name of types after this sentence, otherwise it's nonsense and should be removed.